.github/workflows/integrationChartBump.yaml

```diff
@@ -0,0 +1,87 @@
+name: Integration Chart Promotion
+env:
+  charts: gcloud-sqlproxy datarepo-api datarepo-ui oidc-proxy
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  indivdual_chart_promotion:
+    strategy:
+      matrix:
+        include:
+          - app: datarepo-api
+            repo: "jade-data-repo"
+            filename: "int-and-connected-test-run.yml"
+            jobname: "deploy_test_integration"
+          - app: datarepo-ui
```

Comment:
Oh wow...so you're reaching into the other repo to run the test-e2e action?  That is pretty cool

Reply:
well to edit the chart vals